### PR TITLE
refactor: remove use of `Deno.isatty()`

### LINKED ISSUE #671: Warnings about Deno.stdin in v1.40.1
With the recent update there seems to be some methods used that will be depreciated and produce a very annoying warning.

```warning: Use of deprecated "Deno.isatty()" API. This API will be removed in Deno 2.

Stack trace:
  at Secret.#readChar (https://deno.land/x/cliffy@v1.0.0-rc.3/prompt/_generic_prompt.ts:359:24)
  at Secret.#readKey (https://deno.land/x/cliffy@v1.0.0-rc.3/prompt/_generic_prompt.ts:351:50)
  at Secret.read (https://deno.land/x/cliffy@v1.0.0-rc.3/prompt/_generic_prompt.ts:245:57)
  at Secret.read (https://deno.land/x/cliffy@v1.0.0-rc.3/prompt/secret.ts:96:18)
  at Secret.#execute (https://deno.land/x/cliffy@v1.0.0-rc.3/prompt/_generic_prompt.ts:182:21)
  at async Secret.prompt (https://deno.land/x/cliffy@v1.0.0-rc.3/prompt/_generic_prompt.ts:161:14)
  at async Command.actionHandler (file:///Users/acline/code/admin-cli/main.ts:14:32)
  at async Command.execute (https://deno.land/x/cliffy@v1.0.0-rc.3/command/command.ts:1948:7)
  at async Command.parseCommand (https://deno.land/x/cliffy@v1.0.0-rc.3/command/command.ts:1780:14)

hint: Use `Deno.stdin.isTerminal()`, `Deno.stdout.isTerminal()` or `Deno.stderr.isTerminal()` instead.
hint: It appears this API is used by a remote dependency. Try upgrading to the latest version of that dependency.```

### PR DESCRIPTION
**Pull request by iuioiua**
Closes #671.

### CODE REVIEW

**Comment by c4spar** [comment]
Thx for the fix @iuioiua!

Yeah lets also remove the `rid` property as it's now unused.

**Comment by iuioiua** [comment]
Ok. I've removed all instances of the `rid` property. IIUC, this is a breaking change.

File: prompt/deps.ts [hunk 1]
```diff
@@ -8,9 +8,10 @@ export {
...
   yellow,
-} from "https://deno.land/std@0.196.0/fmt/colors.ts";
+} from "https://deno.land/std@0.214.0/fmt/colors.ts";
 export {
   dirname,
   join,
   normalize,
-} from "https://deno.land/std@0.196.0/path/mod.ts";
+} from "https://deno.land/std@0.214.0/path/mod.ts";
+export type { Reader } from "https://deno.land/std@0.214.0/io/types.ts";
```
Comment (c4spar) [inline prompt/deps.ts:17]: I would prefere to upgrade dependencies in a seperate PR. Also, the exported type `Reader` seems to be unused.

**Review by c4spar — APPROVED** [review]
Thx @iuioiua, LGTM!

**Comment by PrpAcline** [comment]
Thank you! 🎊

**Comment by rubenfiszel** [comment]
Can we do a new release please :)

**Pull request merged**